Tests/Unit/NetworkServer/JsonHandlers/LnsStationConfigurationTests.cs

```diff
@@ -256,6 +256,125 @@ public void WriteRouterConfig()
             Assert.Equal(JsonUtil.Minify(ValidRouterConfigMessage), actual);
         }
 
+        [Fact]
+        public void WriteRouterConfigWithBcning()
+        {
+            var ValidStationConfigurationWithBcning =
+            GetTwinConfigurationJson(new[] { new NetId(1) },
+                                     new[] { (new JoinEui(ulong.MinValue), new JoinEui(ulong.MaxValue)) },
+                                     "EU863",
+                                     "sx1301/1",
+                                     (new Hertz(863000000), new Hertz(870000000)),
+                                     new[]
+                                     {
+                                         (SF11, BW125, false),
+                                         (SF10, BW125, false),
+                                         (SF9 , BW125, false),
+                                         (SF8 , BW125, false),
+                                         (SF7 , BW125, false),
+                                         (SF7 , BW250, false),
+                                     },
+                                     includeBcning: true,
+                                     flags: RouterConfigStationFlags.NoClearChannelAssessment | RouterConfigStationFlags.NoDutyCycle | RouterConfigStationFlags.NoDwellTimeLimitations) ;
+
+        var ValidRouterConfigMessageWithBcning = JsonUtil.Strictify(@"{
```

Comment:
Nit: adding /*lang=json*/ helps the editor understand the content of the text. It is a Visual Studio suggestion :)

```suggestion
        var ValidRouterConfigMessageWithBcning = JsonUtil.Strictify(/*lang=json*/ @"{
```